pkg/cloud/services/compute/instance.go

```diff
@@ -514,7 +524,9 @@ func (s *Service) DeleteInstance(eventObject runtime.Object, instanceName string
 		return err
 	}
 	if len(instanceInterfaces) < 1 {
-		return s.deleteInstance(eventObject, instance.ID)
+		mc := metrics.NewMetricPrometheusContext("server", "delete")
```

Comment:
Please drop the metric here, we already have it one layer below. Let's do this around direct OpenStack API calls